specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -0,0 +1,8387 @@
+{
```

Comment:
In the future, please consider copying over the last production API version as the first commit of the PR. It makes viewing the deltas viable in GH.